### PR TITLE
[ntsc-1.2] Match z_actor_dlftbls.c

### PR DESCRIPTION
**Pull request by cadmic**

### CODE REVIEW

File: src/code/z_actor_dlftbls.c [hunk 1]
```diff
@@ -98,19 +98,47 @@ void ActorOverlayTable_LogPrint(void) {
...
     Fault_Printf("actor_dlftbls %u\n", gMaxActorId);
     Fault_Printf("No. RamStart- RamEnd cn  Name\n");
+#endif
 
     for (i = 0, overlayEntry = &gActorOverlayTable[0]; i < gMaxActorId; i++, overlayEntry++) {
         overlaySize = (uintptr_t)overlayEntry->vramEnd - (uintptr_t)overlayEntry->vramStart;
-        if (overlayEntry->loadedRamAddr != NULL) {
-            Fault_Printf("%3d %08x-%08x %3d %s\n", i, overlayEntry->loadedRamAddr,
-                         (uintptr_t)overlayEntry->loadedRamAddr + overlaySize, overlayEntry->numLoaded,
+        ramStart = (u32)overlayEntry->loadedRamAddr;
```
Comment (Dragorn421) [inline src/code/z_actor_dlftbls.c:126]:
```suggestion
        ramStart = (uintptr_t)overlayEntry->loadedRamAddr;
```

File: src/code/z_actor_dlftbls.c [hunk 2]
```diff
@@ -98,19 +98,47 @@ void ActorOverlayTable_LogPrint(void) {
 void ActorOverlayTable_FaultPrint(void* arg0, void* arg1) {
     ActorOverlay* overlayEntry;
     u32 overlaySize;
+    uintptr_t ramStart;
+    uintptr_t ramEnd;
+    u32 offset;
```
Comment (Dragorn421) [inline src/code/z_actor_dlftbls.c:103]: does
```suggestion
    ptrdiff_t offset;
```
match? (signed type)
if yes that'd be prefered
if not, oh well :shrug:

Comment (cadmic) [reply src/code/z_actor_dlftbls.c:103]: should overlaySize ideally be `size_t` too?

Comment (cadmic) [reply src/code/z_actor_dlftbls.c:103]: ptrdiff_t does not appear to work: https://decomp.me/scratch/UTc4m

Comment (Dragorn421) [reply src/code/z_actor_dlftbls.c:103]: oh well

> should overlaySize ideally be `size_t` too?

sure, if it matches

Comment (cadmic) [reply src/code/z_actor_dlftbls.c:103]: it does not, int vs long thing I guess

**Review by Dragorn421 — APPROVED** [review]

**Review by krm01 — APPROVED** [review]

**Pull request merged**